.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
```

Comment:
is this file relevant for the PR?

Reply:
It is relevant in the sense that it automatically sets the right/consistent editor configurations. But this could be moved in a separate PR of course